gnu-linux/index.md

```diff
@@ -3,6 +3,12 @@ layout: page
 title: GNU/Linux
 ---
 
+# First Steps
+There are many different linux distros, but this guide is going to cover Ubuntu, which is one of the most common ones. The first thing you're going to need is the `build-essential` package, which contains the `gcc` and `g++` compilers, as well as the common build tool `make`. In order to do this, simply run (on the command line) `apt-get install build-essential`.   
+
+# Editors
+If you're planning to write code, you're going to need a way to edit files! If you enjoy working on the command line, we'd recommend either `vim` or `nano`, while if you prefer a GUI, we recommend either [Atom](https://atom.io) or [Brackets](https://brackets.io).
```

Comment:
Hmm this is an interesting direction -- we've never talked about editors on StartHacking, but I guess it is an essential that we've completely missed so far! I think the best way to approach this heavily opinionated topic is use wording along the lines of "most popular" or "common" rather than "we recommend". 

Inserting my personal opinion, I think we should be listing `vim` and `emacs` rather than `vim` and `nano`. I think `emacs` is more popularly used than `nano` for software engineering.

In terms of phrasing the two types of options, I'd start by suggesting Atom or Brackets as common graphical editors (maybe even explain this means using a mouse), and list vim and emacs as more advanced, purely keystroke based editors for when you become comfortable coding.

Minor edits:
*  "GUI" is jargon -- please change it to "graphical interface" or something similar

Reply:
Yup, I agree. `nano` is just the second editor that came to mind as I was writing this. Would it potentially make sense to make a separate page talking about editors, as the vast majority of are cross-platform?